etc/init.d/orchestrator-agent.bash

```diff
@@ -26,6 +26,11 @@ DESC="orchestrator-agent: MySQL management agent"
 PIDFILE=/var/run/$NAME.pid
 SCRIPTNAME=/etc/init.d/$NAME
 
+# This files can be used to inject pre-service execution
+# scripts, such as exporting variables or whatever. It's yours!
+[ -f /etc/default/orchestrator-agent ] && . /etc/default/orchestrator-agent
```

Comment:
Very good. If we're down this rabbit hole, let's also add `/etc/profile.d/orchestrator-agent` to the party.

Reply:
/etc/profile.d/orchestrator-agent
also added see here https://github.com/Slach/orchestrator-agent/blob/reanimate_api/etc/init.d/orchestrator-agent.bash#L33